### PR TITLE
Sprint 22 selenium tests

### PR DESCRIPTION
**Pull request by andyh0311**

### CODE REVIEW

File: selenium-testing/src/main/java/com/testng/selenium/v1/App.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package com.testng.selenium.v1;
+
+/**
+ * Hello world!
+ *
+ */
+public class App
```
Comment (weskubo-cgi) [inline selenium-testing/src/main/java/com/testng/selenium/v1/App.java:7]: Is this class required?

File: selenium-testing/pom.xml [hunk 2]
```diff
@@ -0,0 +1,70 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>com.testng.selenium</groupId>
```
Comment (weskubo-cgi) [inline selenium-testing/pom.xml:6]: See other comments on code packages.

File: selenium-testing/src/test/java/com/testng/selenium/v1/BaseTest.java [hunk 3]
```diff
@@ -0,0 +1,101 @@
+package com.testng.selenium.v1;
```
Comment (weskubo-cgi) [inline selenium-testing/src/test/java/com/testng/selenium/v1/BaseTest.java:1]: Normally, your package would be relevant to the project you're working on, not the framework used.

E.g. ca.bc.gov.ecc.ofm.selenium.v1 might be a good choice

File: selenium-testing/config.properties.template [hunk 4]
```diff
@@ -0,0 +1,7 @@
+portal_url=https://ofm-frontend-test-e1800b-dev.apps.silver.devops.gov.bc.ca/login/
```
Comment (weskubo-cgi) [inline selenium-testing/config.properties.template:1]: At some point, you might want to add a quick README.md explaining how to do things like setup the local config.properties and run tests.

**Review by weskubo-cgi — CHANGES_REQUESTED** [review]

File: selenium-testing/src/main/java/com/testng/selenium/v1/App.java (hunk 1 above)
Comment (andyh0311) [reply selenium-testing/src/main/java/com/testng/selenium/v1/App.java]: Done: https://github.com/bcgov/ecc-ofm/pull/319/commits/e4c38d2ffd83f1fc0cd1e233099a8f907635a9ac

File: selenium-testing/pom.xml (hunk 2 above)
Comment (andyh0311) [reply selenium-testing/pom.xml]: Changed in pom.xml: https://github.com/bcgov/ecc-ofm/pull/319/commits/5247c4beed17cdb26a54db8c56bc613625051a32

File: selenium-testing/src/test/java/com/testng/selenium/v1/BaseTest.java (hunk 3 above)
Comment (andyh0311) [reply selenium-testing/src/test/java/com/testng/selenium/v1/BaseTest.java]: Refactored: https://github.com/bcgov/ecc-ofm/pull/319/commits/180b5b17bf2e8dc5662e9373deea7738980cf48a

File: selenium-testing/config.properties.template (hunk 4 above)
Comment (andyh0311) [reply selenium-testing/config.properties.template:1]: Done: https://github.com/bcgov/ecc-ofm/pull/319/commits/bce2de80d5de06d049add83578bcdae3a627059e

File: selenium-testing/README.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+**`Step 1`** Download JDK 21
+
+**`Step 2`** Download the Eclipse IDE
+
+**`Step 3`** On the Eclipse marketplace, install the **TestNG for Eclipse** plugin
+
+## Run tests
+
+First, rename the file `config.properties.template` to `config.properties`, then fill in the missing details with base64 string encoded values
```
Comment (weskubo-cgi) [inline selenium-testing/README.md:15]: You probably want to change this step to copy and name as. If you rename it'll show up as a delete in git.

Comment (andyh0311) [reply selenium-testing/README.md]: Ohh I see: https://github.com/bcgov/ecc-ofm/pull/319/commits/9414a142742a844d2b6f841143a9e6c034e190a6

**Review by weskubo-cgi — APPROVED** [review]

**Pull request merged**